packages/hub/src/components/EntityInfo.tsx

```diff
@@ -1,16 +1,28 @@
 import { FC } from "react";
 
 import { UsernameLink } from "./UsernameLink";
+import { modelRoute } from "@/routes";
+import Link from "next/link";
 
 // works both for models and for definitions
 export const EntityInfo: FC<{ username: string; slug: string }> = ({
   username,
   slug,
 }) => {
   return (
-    <div>
-      <span className="text-xl font-bold">{slug}</span> by{" "}
-      <UsernameLink username={username} />
+    <div className="flex justify-between w-full">
+      <div className="flex items-center mr-3 group cursor-pointer">
+        <Link
+          className="text-xl font-medium text-blue-600 group-hover:underline"
+          href={modelRoute({ slug, username })}
+        >
+          {slug}
+        </Link>
```

Comment:
I don't think it should be a link:
- I mostly see it on model pages and it looks like it should do something interesting, but it doesn't do anything
- on revision pages, there's a separate "Go to latest version" link
- also, on definition pages, it's broken; /users/berekuk/relative-values/aggregate sends me to non-existent /users/berekuk/models/aggregate.

Reply:
I like the link aspect. Will fix the relative values part, didn't see/know that.

It's a link on Github. My guess is that we'll later have more tabs, besides just the Revision tab. 

As a user, I'd expect that clicking this would do *something*, and would want some easy way of going "home"

That said, there's definitely more work to standardize entities (RVs + Models)